api/v1alpha1/ocivalidator_types.go

```diff
@@ -47,18 +46,7 @@ type OciRegistryRule struct {
 }
 
 func (r OciRegistryRule) Name() string {
-	var artifacts strings.Builder
-	artifacts.WriteString("[")
-	l := len(r.Artifacts)
-	for i, a := range r.Artifacts {
-		artifacts.WriteString(a.Ref)
-		if i < l-1 {
-			artifacts.WriteString(", ")
-		}
-	}
-	artifacts.WriteString("]")
-
-	return fmt.Sprintf("%s/%s", r.Host, artifacts.String())
+	return fmt.Sprintf("%s/%v", r.Host, len(r.Artifacts))
```

Comment:
Sorry for the constant stream of nits, but I would use `"%s/%d"` here. Although according to [this](https://stackoverflow.com/questions/41159169/why-not-use-v-to-print-int-and-string) it doesn't really matter. Thought you might find that interesting though.